conda-forge.yml

```diff
@@ -1,3 +1,5 @@
+bot:
+  inspection: update-grayskull
```

Comment:
This is just a recommendation.  I can remove the last commit if you want.  But this will automatically update the dependencies for each new release by using the [grayskull](https://github.com/conda/grayskull) tool.